notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb

```diff
@@ -810,7 +756,7 @@
         "Now define the command-line arguments for your custom training container:\n",
         "\n",
         "- `args`: The command-line arguments to pass to the executable that is set as the entry point into the container.\n",
-        "  - `--model-dir` : For our demonstrations, we use this command-line argument to specify where to store the model artifacts.\n",
+        "  - `--model-dir` : For our demonstrations, use this command-line argument to specify where to store the model artifacts.\n",
```

Comment:
"For these demonstrations..." (don't use 1st person plural pronouns)

Reply:
Did required changes.